routes/login/login.spec.js

```diff
@@ -653,24 +665,8 @@ questionsAmounts.map(amountResponse => {
           .post(amountResponse.url)
           .use(withCSRF(cookie, csrfToken))
           .send({
-            taxReturnYear: '2018',
-            taxReturnAmount: goodAmount,
-            redirect: '/start',
-          })
-        expect(response.statusCode).toBe(302)
-        expect(response.headers.location).toEqual('/start')
-      })
-    })
-
-    const goodYears = ['2018', '2017']
```

Comment:
Was there a reason we originally tested against 2 years?

Reply:
It was rewritten from an earlier check where only 2 values were accepted.

We accept tons of year values so these two are arbitrary.

Reply:
Okay, you win.